scripts/report-references-to-old-versions.js

```diff
@@ -13,14 +13,20 @@ const FILES_TO_IGNORE = [
     '../microsoft-edge/devtools-guide-chromium/experimental-features/index.md'
 ];
 // This script attempts to find the current edge release version by looking at this page and finding the first version header occurrence.
-const RELEASE_NOTES_PAGE = 'https://docs.microsoft.com/deployedge/microsoft-edge-relnote-stable-channel';
+const RELEASE_NOTES_PAGE = 'https://learn.microsoft.com/deployedge/microsoft-edge-relnote-stable-channel';
 
+// This is the list of regular expressions we use to find references to Edge versions.
```

Comment:
```suggestion
// This is the list of regular expressions we use to find references to Microsoft Edge versions.
```
this repo usually uses stricter form